src/collections/programs/hacktoberfest-2020/index.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Hacktoberfest 2020"
+title: "Hacktoberfest 2021"
```

Comment:
Do we mean to update the 2020 version or leave it in place?

Reply:
It'll be better to just update it.

Reply:
We're sort of half way between 2020 and 2021 in this PR. Understanding the state of your sickly brain (completely understandable) at this point in time, I'll remind you to "scroll down" as you think this through.

Reply:
(I made a funny right there, in case you missed it)

Reply:
actually, I didn't change the permalink because I kept the name of the folder the same 😅.